msi-builder/postinstall.bat

```diff
@@ -0,0 +1,15 @@
+@echo off
+SET InstallDir=%~1
+SET FilePath=%InstallDir%\os\finch.yaml
+
+if exist "%FilePath%" (
+    powershell -Command "$installPath = '%InstallDir%'.Replace('\', '/'); (Get-Content '%FilePath%') -replace '__INSTALLFOLDER__', $installPath | Set-Content '%FilePath%'"
```

Comment:
I don't see where `__INSTALLFOLDER__` is set. Should there be a manual step to add it to `os\finch.yaml` before starting to build the installer?

Reply:
Thanks. We should probably still add that to the README just in case we ever need to do a manual build

Reply:
Thanks. I will update the ReadMe with the auto build PR.